rust/agama-dbus-server/src/questions.rs

```diff
@@ -58,22 +69,19 @@ impl LuksQuestionObject {
     pub fn set_password(&mut self, value: &str) -> () {
         self.0.password = value.to_string();
     }
-
-    #[dbus_interface(property)]
-    pub fn attempt(&self) -> u8 {
```

Comment:
Is it intentional to discard `attempt`?

Reply:
yeap, as I think it is not generic password specific stuff. So my plan now is to move it do data as ro data and if we need it more often, we can add WithAttempt mixin for questions that needs to capture it.